docs/documentation/pack-docs.md

```diff
@@ -55,13 +55,15 @@ For larger packs that provide at least one end-to-end use case you should create
 ### README structure
 - If the pack is a ***beta*** pack, add the following at the very beginning of the pack README file:
 _Note: This is a beta pack, which lets you implement and test pre-release software. Since the pack is beta, it might contain bugs. Updates to the pack during the beta phase might include non-backward compatible features. We appreciate your feedback on the quality and usability of the pack to help us identify issues, fix them, and continually improve._
-* A short paragraph connecting real-life situations to the pack use cases.
-* A "What does this pack do?" section, explaining point-by-point the capabilities of the pack or the main playbook of the pack.
-* (Optional): a sentence or two detailing the contents of the pack.
-* Mostly relevant for use-case packs that contain multiple playbooks: a reference to the README of the **main playbook** of the pack that contains the playbook logic. Add the following:
+- If the pack is adopted, add the correct text as specified on the [Adopt-a-Pack page](../partners/adopt.md#text-for-the-pack).
+
+1. A short paragraph connecting real-life situations to the pack use cases.
+1. A "What does this pack do?" section, explaining point-by-point the capabilities of the pack or the main playbook of the pack.
+1. (Optional): a sentence or two detailing the contents of the pack.
+1. Mostly relevant for use-case packs that contain multiple playbooks: a reference to the README of the **main playbook** of the pack that contains the playbook logic. Add the following:
```

Comment:
In the beginning, we have bullets and then enumerations. Is this on purpose? There is no clear separation between them I think.

Reply:
Yeah I was also confused by what was there. I changed it, better @ShahafBenYakir ?